gulp/tasks/gen-service-worker.js

```diff
@@ -86,13 +85,29 @@ gulp.task('gen-service-worker', () => {
         rootDir + '/fonts/roboto/Roboto-Bold.ttf',
         rootDir + '/images/card_media_player_bg.png',
       ],
-      runtimeCaching: [{
-        urlPattern: /\/static\/translations\//,
+      // Rules are proceeded in order and negative per-domain rules are not supported.
+      runtimeCaching: [
+        { // Cache translations on first access
+          urlPattern: '/static/translations/*',
         handler: 'cacheFirst',
-      }, {
-        urlPattern: RegExp('^[^/]*/' + fallbackList + '.'),
+        },
+        { // Get static, api, and local (and home-assistant-polymer in dev mode) from network.
+          // Use cache only if Home Assistant server is unreachable.
+          urlPattern: '/(home-assistant-polymer|static|api|local)/*',
```

Comment:
We should not get /api requests from cache.

Reply:
It's `networkOnly` now.

Reply:
Ah, I was thrown off by the comment.